test/helpers/SigUtils.sol

```diff
@@ -1,8 +1,8 @@
-// SPDX-License-Identifier: GPL-2.0-or-later
+// SPDX-License-Identifier: MIT
```

Comment:
There were files with MIT, files with GPL-2.0-or-later, and files with UNLICENSED. I went for MIT

Reply:
Don't know if it matters but in [metamorpho](https://github.com/morpho-org/metamorpho/blob/main/test/forge/helpers/SigUtils.sol) and [morpho-blue-bundlers](https://github.com/morpho-org/morpho-blue-bundlers/blob/main/test/forge/helpers/SigUtils.sol) this specific file for instance is GPL-2.0-or-later.

Reply:
yes we should go for GPL2